atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -116,8 +116,8 @@ def download_sources(self, urls, insecure=False, download_dir=SRPMS_DOWNLOAD_DIR
 
         req_session = get_retrying_requests_session()
         for url in urls:
-            download_url(url, dest_dir, insecure=insecure,
-                         session=req_session)
+            download_url(url['url'], dest_dir, insecure=insecure,
```

Comment:
Maybe we can call it source(s). `url['url']` seems weird. Also docstring for url param is wrong

Reply:
ok